wordpress-mobile/WordPress-Android
# Enable My Site Dynamic Cards remote feature flag

This PR enables the remote flag for the My Site Dynamic Cards feature.

For more information about this feature, check its [project board](https://github.com/orgs/wordpress-mobile/projects/79) or this internal ref: pbMoDN-214-p2

To test:

1. Clear app data.
2. Open the app.
3. Make sure you don't manually activate the feature flag.
4. Create a new site and verify that the new Quick Start appears.

PR submission checklist:
